README.md

```diff
@@ -30,6 +30,12 @@ Built with clean architecture and domain driven design with clear seperation of
 
 ## 💡 Get started
 
+1. Download the configuration file from the Firebase Console (the file is called google-services.json). Add this file into the android/app directory within your Flutter project.
+
+2. Download the configuration file from the Firebase Console (the file is called GoogleService-Info.plist). Add this file via [this](https://firebase.flutter.dev/docs/installation/ios#installing-your-firebase-configuration-file) instruction.
+
+3. Create file (the file is called firebase_config.js). This file should contains the configuration for firebase as you can see `firebase_config.js.example`.
```

Comment:
1. Can you add the relative path `web/script/firebase_config.js.example`?
2. Consider renaming the folder `web/script/` as `web/config/`
3. Please add config code snippet from `firebase_config.js.example` in the README

Reply:
Sure